src/CMakeLists.txt

```diff
@@ -88,6 +88,7 @@ set(keepassx_SOURCES
         gui/DatabaseTabWidget.cpp
         gui/DatabaseWidget.cpp
         gui/DatabaseWidgetStateSync.cpp
+        gui/DownloadIcon.cpp
```

Comment:
Can we name this file (and class) "IconDownloader" to comply with the associated dialog?